draft-barnes-sframe-mls.md

```diff
@@ -64,8 +64,42 @@ properties apply to real-time media as well.  In the remainder of this document,
 we define how to use the secrets produced by MLS to generate the keys required
 by SFrame.
 
-[[ OPEN ISSUE: We could define an MLS extension that would provide negotiation
-of SFrame parameters, notably the ciphersuite and the value E defined below. ]]
+# SFrame Parameter Negotiation
+
+In order to interoperate, the sender and receiver(s) of an SFrame payload need
+to agree on two parameters:
+
+* The SFrame ciphersuite
+* The number of bits `E` used to signal the epoch
+
+These parameters can be negotiated in MLS using the `sframe_parameters`
+extension.  An MLS participant advertises its supported ciphersuites in its
+KeyPackage.  The creator of the group chooses the values of these parameters for
+the group (possibly based on a set of KeyPackages) and advises them to new
+joiners in Welcome messages.
```

Comment:
wondering if we need more text on how the creator chooses the right ciphersuite from the list provided . Thinking, how to get to a eventual strong ciphersuite for the group when the members join and not have the weakest one define the ciphersuite for the group (say if such a member joins early and creator let's him)

May be we need to list the ciphersuites in this document allowed to prevent such downgrades ?

Reply:
Definitely a good point. I guess we could mimic what we do in the MLS protocol doc and publish alist of ciphersuites in the IANA considerations section.

Reply:
The list of ciphersuites is in the SFrame specification, and the selection is "whatever the group creator decides based on the capabilities expressed in the KeyPackages of the initial joiners (and expectations of later joiners)".  I can provide some notes on the latter.